CMakeLists.txt

```diff
@@ -1,3 +1,8 @@
+set(VERSION_ROS2 "ROS2" )
+set(VERSION_HUMBLE "humble" )
+set(ROS_EDITION ${VERSION_ROS2}  )
+set(HUMBLE_ROS ${ROS_HUMBLE} )
```

Comment:
This part probably wasn't meant to be hardcoded - but otherwise it all seems to work well.